frameworks/Python/weppy/weppy-nginx-uwsgi.dockerfile

```diff
@@ -1,8 +1,8 @@
-FROM python:2.7.14
+FROM python:3.6.6-stretch
```

Comment:
Did you mean to bump major version here? If so would you mind updating the notes section in https://github.com/TechEmpower/FrameworkBenchmarks/blob/084e0f47bf0704d8bc3e82c1bab5bb9268daeab9/frameworks/Python/weppy/benchmark_config.json#L90